pufferlib/ocean/tetris/tetris.h

```diff
@@ -0,0 +1,533 @@
+#include <stdlib.h>
+#include <math.h>
+#include <assert.h>
+#include <unistd.h>
+#include <limits.h>
+#include <string.h>
+#include "raylib.h"
+#include "tetrominoes.h"
+#define min(a, b) (((a) < (b)) ? (a) : (b))
+#define max(a, b) (((a) > (b)) ? (a) : (b))
+
+#define HALF_LINEWIDTH 1
+#define SQUARE_SIZE 32
+#define MAX_STEPS 400
+
+const int REWARDS[5] = {0, 40, 100, 300, 1200};
+const int REWARD_INVALID_ACTION = -100;
+
+typedef struct Log {
+    float perf;
+    float score;
+    float episode_return;
+    float avg_combo;
+    float episode_length;
+    float lines_deleted;
+    float n;
+} Log;
+
+typedef struct Client {
+    int total_cols;
+    int total_rows;
+    int ui_rows;
+    int deck_rows;
+    int preview_rows;
+    int preview_target_rotation;
+    int preview_target_col;
+} Client;
+
+typedef struct Tetris {
+    Client* client;
+    Log log;
+    float* observations;
+    float* actions;
```

Comment:
Change to int! Will not train as is because Python is writing the bits of an int to a float